README.md

```diff
@@ -22,7 +22,8 @@ We welcome pull requests for bugs described in our issue tracker. Please see our
 * [Developer Documentation](https://github.com/otwcode/otwarchive/wiki)
 * [Commit Policy](https://github.com/otwcode/otwarchive/wiki/Commit-policy)
 
-We do not have a public chat, but you are welcome to contact us at otw-coders@transformativeworks.org if you have any questions.
+We do not have a public chat, but you are welcome to contact us at otw-coders@transformativeworks.org if you have any questions.  
+We grant your Jira account permissions for commenting on, assigning, and transitioning issues [after you create your first pull request](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md#workflow).
```

Comment:
It looks like this is meant to be a different paragraph than the part about the chat, so I think we need a blank line between the two lines.

Reply:
It's meant to be a linebreak but not a different paragraph, like this.  
Second sentence.

So my question is whether we want that formatting or if a new paragraph would be more standard. I'd be fine with either.

Reply:
Let's make it a new paragraph.